src/ray/common/constants.h

```diff
@@ -5,7 +5,10 @@
 #include <stdint.h>
 
 /// Length of Ray full-length IDs in bytes.
-constexpr int64_t kUniqueIDSize = 20;
+constexpr size_t kUniqueIDSize = 20;
+
+/// Length of Job ID in bytes.
+constexpr int64_t kJobIDSize = 4;
```

Comment:
Since all id sizes will be different, I think it's not very useful to define a lot of constants here. Maybe just use something like `JobID::Size()`.